### DISCUSSION TITLE
Is  lscr.io/linuxserver/pyload-ng recommended?

### DISCUSSION THREAD

**Discussion by gerroon** [pyload/pyload#4530] [Help] [answered]
Hi

I set this up long time ago and I have been using it since then. I am wondering if you recommend using this Docker setup over the repo one.

edit: Repo one doesn't seem to exist?

```
docker pull pyload/pyload:alpine
Error response from daemon: manifest for pyload/pyload:alpine not found: manifest unknown: manifest unknown

```

**Comment by GammaC0de** [comment, marked as answer]
The linuxserver docker is surely more active and has better support than I can offer regarding docker.

> Repo one doesn't seem to exist?

No, you build a docker image yourself,
See instruction in https://github.com/pyload/pyload?tab=readme-ov-file#docker-images

**Reply by gerroon** [reply]
Thank you, I appreciate it.
